docs/Get started/FAQ.md

```diff
@@ -4,7 +4,7 @@
 
 Fleet offers managed cloud hosting for [Fleet Premium](https://fleetdm.com/pricing) customers with large deployments.
 
-> While organizations of all kinds use Fleet, from Fortune 500 companies to school districts to hobbyists, we are only currently able to provide cost-effective hosting for deployments larger than 1000 hosts.  (Instead, you can [buy a license](https://fleetdm.com/customers/register) and self-host Fleet Premium with support.)
+> While organizations of all kinds use Fleet, from Fortune 500 companies to school districts to hobbyists, today we are only currently able to provide fully-managed hosting for deployments larger than 300 hosts.  (Instead, you can [buy a license](https://fleetdm.com/customers/register) and self-host Fleet Premium with support.)
```

Comment:
fyi @noahtalerman @lukeheath @alexmitchelliii @zayhanlon  -- lmk if you disagree

Reply:
this works for me, just bearing in mind the minimal cost of running infra for <1000 hosts when quoting @alexmitchelliii